wheel_matrix.yml

```
@@ -91,7 +91,7 @@ packages:
           - tag: cp311
   coverage:
     versions:
-      6.5.0:
+      7.2.7:
```

This needs to be
```suggestion
      7.3.0:
```
now.